cpu/kinetis/Makefile.features

```diff
@@ -2,6 +2,11 @@ FEATURES_PROVIDED += periph_cpuid
 FEATURES_PROVIDED += periph_hwrng
 FEATURES_PROVIDED += periph_gpio
 FEATURES_PROVIDED += periph_gpio_irq
+ifneq (EA,$(KINETIS_SERIES))
```

Comment:
can you please explain why the features `mcg` and `ics` are exclusive for the `EA` series? I mean `ics` is new, but you change the usage of `mcg` here.